dusty/config.py

```diff
@@ -122,9 +123,6 @@ def check_and_load_ssh_auth():
     SSH_AUTH_SOCK environment variable to the current environment.  This allows git clones
     to behave the same for the daemon as they do for the user
     """
-    if os.getenv('SSH_AUTH_SOCK'):
-        logging.info('SSH_AUTH_SOCK already set')
-        return _set_ssh_auth_sock(os.getenv('SSH_AUTH_SOCK'))
```

Comment:
This block is the entrypoint, not the stuff up there. This deleted code was preventing it from doing anything if the envvar had ever been set.